src/main/java/seedu/address/logic/parser/exceptions/WrongDateFormatException.java

```diff
@@ -0,0 +1,14 @@
+package seedu.address.logic.parser.exceptions;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+
+public class WrongDateFormatException extends IllegalValueException {
+
+    public WrongDateFormatException(String message) {
+        super(message);
+    }
+
+    public WrongDateFormatException(String message, Throwable cause) {
+        super(message, cause);
+    }
+}
```

Comment:
I think that there is no need to create a separate exception for wrong date. Instead, parseException should be used. 

According to the code in Parser class, any exceptions or mistakes in the input is returned in a ParseException. In this case, we can save on a class whose implementation is identical to ParseException.

Reply:
Done.